install/kubernetes/helm/istio/charts/pilot/templates/clusterrole.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if .Values.global.rbacEnabled }}
```

Comment:
Why remove this? If this is removed, the `cr` will always be created. But for the case rbac disabled, this seems no use?

Ditto for other places where removing the rbac check condition.

Reply:
Pilot needs the CR to create CRDs, etc. Running as default (when RBAC is disabled) is not enough because default doesn't have permissions for the operations. Therefore binding this CR to the default ServiceAccount resolves the problem.